pkg/project/auth/watch.go

```diff
@@ -158,16 +173,26 @@ func (w *userProjectWatcher) Watch() {
 	}()
 	defer utilruntime.HandleCrash()
 
+	// start by emitting all the `initialProjects`
+	for i := range w.initialProjects {
+		// keep this check here to sure we don't keep this open in the case of failures
+		select {
+		case err := <-w.cacheError:
+			w.emit(makeErrorEvent(err))
+			return
+		default:
+		}
+
+		w.emit(watch.Event{
+			Type:   watch.Added,
```

Comment:
I know this watch isn't normal but is this what happens in normal watches? We get only Added events up to the point we start watching?